### PR TITLE
Add python-coveralls

### PR DESCRIPTION
**Pull request by jakirkham**
Used `conda skeleton pypi` to generate the recipe and made some edits to make it work nicely here at conda-forge.

### CODE REVIEW

File: recipes/python-coveralls/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  version: {{ version }}
+
+source:
+  fn: python-coveralls-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/python-coveralls/python-coveralls-{{ version }}.tar.gz
+  md5: e66878a926b9d5dd04250e1a05080fbd
+
+build:
+  number: 0
+  skip: true  # [win]
```
Comment (jakirkham) [inline recipes/python-coveralls/meta.yaml]: There is no `sh` on Windows so we must skip.

**Comment by jakirkham** [comment]
@z4r @ahxxm, just letting you know we have added `python-coveralls` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) (`conda install -c conda-forge python-coveralls`). The maintenance burden here is light as all testing and releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this repo]( https://github.com/conda-forge/python-coveralls-feedstock ). If you have any interest in being added as a maintainer (to make new releases for instance), I can happily add you to the maintainers list.

**Comment by jakirkham** [comment]
@dan-blanchard, any interest in being a maintainer of `python-coveralls`?

**Comment by ahxxm** [comment]
@jakirkham haven't use `conda` before, if I understand correctly [it](https://conda-forge.github.io/) is an automated subset pypi?

**Comment by dan-blanchard** [comment]
> @dan-blanchard, any interest in being a maintainer of python-coveralls?

I'll stick to just maintaining recipes for things I regularly use.

**Comment by jakirkham** [comment]
> haven't use `conda` before, if I understand correctly [it](https://conda-forge.github.io/) is an automated subset pypi?

@ahxxm, it isn't an automated subset of PyPI yet. Though we are thinking about that ( https://github.com/conda-forge/conda-forge.github.io/issues/28 ).

The point of `conda` (what this is all based around) is to handle some of the more gnarly parts of the Python ecosystem like NumPy, SciPy, things requiring Qt, etc. It also is designed to do this in a way where `sudo` isn't required, which makes it really helpful on clusters for instance. Though, in doing so, it ends up being pretty effective at extending into other domains like Perl, R, C, C++, Fortran, JavaScript, etc. There is a bit of focus in the ecosystem on numeric heavy software. Though it is not limited to that. For some time, [Continuum]( https://www.continuum.io/ ) has handled most of the packaging for us. However, we started to realize that, even though what they provided was really useful, it was not enough.

The point of conda-forge was to largely automate and standardize builds of software in a community friendly way. In the last month and a half, our size has more quadrupled and the number of things packaged has increased by more than an order of magnitude.

Though, some work is required to update versions or change details of a package from time to time. If you would be willing to help make sure we continue to packaging `python-coveralls` in a correct manner, that would be very helpful. Also, it is good for us to stay informed on new releases. So if you could let us know when you are planning to cut a new release, that would be great.

**Comment by jakirkham** [comment]
> > @dan-blanchard, any interest in being a maintainer of python-coveralls?

> I'll stick to just maintaining recipes for things I regularly use.

Sorry to ping you unnecessarily, @dan-blanchard. I thought this might be something you are still using with CI so figure I should let you know. To avoid unnecessary pings in the future, could you please provide us a list of what you do care about in this issue ( https://github.com/conda-forge/staged-recipes/issues/139 )? That way we can try to ping you when something interesting comes along, but otherwise improve the SNR for you.

**Pull request merged**